src/main/java/seedu/address/model/student/StudentId.java

```diff
@@ -9,9 +9,10 @@
  */
 public class StudentId {
 
-    public static final String MESSAGE_CONSTRAINTS = "Student ID should only contain "
-            + "alphanumeric characters and should not start or end with spaces.\"";
-    public static final String VALIDATION_REGEX = "[A-Za-z0-9]+";
+    public static final String MESSAGE_CONSTRAINTS = "Student ID should be in the format: A followed by seven digits"
+            + "and an uppercase character and should not start or end with spaces.\"";
+
+    public static final String VALIDATION_REGEX = "^A\\d{7}[A-Z]$";
```

Comment:
I like how the Student ID is now formatted! It must start with an 'A', followed by exactly seven digits and end with an uppercase letter!